addons/zeus/functions/fnc_bi_moduleCurator.sqf

```diff
@@ -197,8 +200,8 @@ if (_activated) then {
         } foreach (synchronizedobjects _logic);
         _addons call bis_fnc_activateaddons;
 
-        // Added by ACE_zeus to delay ascension messages and bird code (so that settings can be changed)
-        [{_this spawn DFUNC(moduleCuratorDelay);},[_logic,_player]] call EFUNC(common,execNextFrame);
```

Comment:
True, it was originally that way to get back into unscheduled environment. Just forgot to change it now that the context has changed.